Scalar/CommandLine/CloneVerb.cs

```diff
@@ -333,7 +333,7 @@ private Result GitClone()
                 git.SetInLocalConfig($"core.sparseCheckoutCone", "true");
 
                 this.fileSystem.CreateDirectory(Path.Combine(this.enlistment.DotGitRoot, "info"));
-                this.fileSystem.WriteAllText(Path.Combine(this.enlistment.DotGitRoot, "info", "sparse-checkout"), "/*\n!/*/*");
+                this.fileSystem.WriteAllText(Path.Combine(this.enlistment.DotGitRoot, "info", "sparse-checkout"), "/*\n!/*/");
```

Comment:
do we want a trailing \n here ??

Reply:
I just ran a local test clone and the file is created without the final NL.

Reply:
Sorry that my commits aren't great here. The final commit uses `git sparse-checkout init --cone` directly, which skirts this issue. Thanks for your attention to detail!